code-reviews/example3/adventure_game.py

```diff
@@ -0,0 +1,79 @@
+# An interactive adventure game!
+import random
+
+def print_welcome():
+    print("Welcome to the adventure game! 🛡")
+    print("Maintain your health while finding food 🥕")
+    print("But watch out for the wild animals! 🦊")
+    print("")
+
+def print_menu():
+    print("1. Explore")
+    print("2. Check status")
+    print("3. Rest")
+    print("4. Quit game")
+
+def explore():
+    global health
+    global food
+    event = random.randint(1, 3)
```

Comment:
Interesting game! I sort of wish it would be more interactive than random though.

Reply:
Agreed, random isn't the best. What would you recommend for this?